cli/azd/pkg/project/resources.go

```diff
@@ -79,6 +82,11 @@ func (r *ResourceConfig) MarshalYAML() (interface{}, error) {
 		if err != nil {
 			return nil, err
 		}
+	case ResourceTypeDbMySQL:
+		err := marshalRawProps(raw.Props.(MySQLProps))
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Is it necessary to add other resource type like: ResourceTypeDbRedis, ResourceTypeDbPostgres, ResourceTypeDbMong?

Reply:
Could you please add unit test for `MarshalYAML` and `UnmarshalYAML`?